bugbug/models/regressor.py

```diff
@@ -127,9 +128,16 @@ def __init__(
                 ("union", ColumnTransformer(column_transformers)),
             ]
         )
-
-        self.clf = xgboost.XGBClassifier(n_jobs=utils.get_physical_cpu_count())
-        self.clf.set_params(predictor="cpu_predictor")
+        self.calibration = True
+        if self.calibration:
+            base_clf = xgboost.XGBClassifier(n_jobs=utils.get_physical_cpu_count())
+            base_clf.set_params(predictor="cpu_predictor")
```

Comment:
This common logic could be done outside the calibration condition.

Reply:
makes sense!